src/scss/theme/_button.scss

```diff
@@ -36,5 +36,12 @@
   }
 }
 
+// Back to all events or news link.
+.more-link {
+  a {
+    @include box-shadow('shallow');
+  }
+}
```

Comment:
Is this really a button and do you want the drop shadow everywhere there is a `more-link` class?

Reply:
It is really a button but not in HTML. Maybe I should change the HTML? `<div class="more-link"><a href="/news">See All News</a></div>`